### PR TITLE
keep render headerLeft component on first screen

### PR DESCRIPTION
**Pull request by jemmyphan**
rendering headerLeft with component when route index === 0 isn't working

### CODE REVIEW

File: src/views/Header/Header.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ class Header extends React.PureComponent {
       return options.headerLeft;
     }
 
-    if (props.scene.index === 0) {
+    if (props.scene.index === 0 && options.headerLeft === undefined) {
```
Comment (vonovak) [inline src/views/Header/Header.js]: using `!options.headerLeft` would be better since it also includes `null`

**Comment by vonovak** [comment]
hi @jemmyphan can you add a little more background? What is the motivation for such change? It that a breaking change?

**Comment by ericvicenti** [comment]
Look at line 133. If you provide an <Element /> to headerLeft instead of a React Component class, you should be good to go with the current header implementation.

Perhaps we should add a warning for this.

**Comment by jemmyphan** [comment]
@ericvicenti providing <Element /> to headerLeft is working, but using Element instead of Component class will omit the props getting passed (... onPress)
@vonovak I have to use the props that passed to headerLeft, I use undefined because there is null checking on line 164 already. 😅

**Pull request closed without merging**